common/oatbox/user/UserLanguageService.php

```diff
@@ -38,7 +38,7 @@ class UserLanguageService extends ConfigurableService implements UserLanguageSer
      */
     public function getDefaultLanguage()
     {
-        return DEFAULT_LANG;
+        return $this->getOption(self::OPTION_DEFAULT_LANGUAGE, DEFAULT_LANG);
```

Comment:
Be extra careful with this change. The method is by the next method `getDataLanguage`. 
So in UDIR case you will set data language to Norwegian and this will break all the the resources label rendering and the properties. 

Consider using the new default language definition only for the interface language.

Reply:
We should only override the interface language in this case

Reply:
I have rechecked how this work after the [comment in the tao-udir pull request](https://github.com/oat-sa/extension-tao-udir/pull/142#discussion_r771368566) and I've reached the same conclusion :sweat_smile: I'll change this to make the interface language configurable instead of this one.

Reply:
Making `getInterfaceLanguage()` configurable, reverting `getDefaultLanguage()` to its former behavior, in 4dbf4834ab3fa856658c763625d7c74b7ccd77d8

Reply:
This was restored to its former behavior (see  [here](https://github.com/oat-sa/generis/blob/feature/ADF-842-configurable-user-default-language/common/oatbox/user/UserLanguageService.php#L39)), marking this thread as solved :slightly_smiling_face: